zipline/testing/core.py

```diff
@@ -1156,6 +1158,72 @@ def create_empty_splits_mergers_frame():
     )
 
 
+def make_alternating_boolean_array(shape, first=True):
```

Comment:
I might call this `first_value` for clarity.  Seeing `first=True` makes me think it's a flag.